docs/apache-airflow-providers-imap/connections/imap.rst

```diff
@@ -60,4 +65,4 @@ For example:
 
 .. code-block:: bash
 
-   export AIRFLOW_CONN_IMAP_DEFAULT='imap://username:password@myimap.com:993'
+   export AIRFLOW_CONN_IMAP_DEFAULT='imap://username:password@myimap.com:993?use_ssl=true'
```

Comment:
Since the default is true, maybe we should show two examples, one for the default (SSL) case, another for the non-SSL case.

Reply:
Sure! Let me add one.

Reply:
Let me know what you think efa4628